docs/configuration.md

```diff
@@ -1857,6 +1857,61 @@ Apart from these, the following properties are also available, and may be useful
     driver using more memory.
   </td>
 </tr>
+<tr>
+  <td><code>spark.scheduler.listenerbus.eventqueue.shared.capacity</code></td>
+  <td><code>spark.scheduler.listenerbus.eventqueue.capacity</code></td>
+  <td>
+    Capacity for shared event queue in Spark listener bus, must be greater than 0.
+    Unless otherwise specified it uses <code>spark.scheduler.listenerbus.eventqueue.capacity</code>.
```

Comment:
I think this is covered already by the fact that this is shown as the default here. You can remove it everywhere.

Reply:
Done